cmd/import.go

```diff
@@ -67,10 +68,10 @@ Json Format:
 			fileType := getOrInferFileType(fileName)
 			switch fileType {
 			case "csv":
-				err := inout.ImportEventTypesCsv(svixClient, reader, force)
+				err := inout.ImportEventTypesCsv(context.Background(), svixClient, reader, force)
 				printer.CheckErr(err)
 			default:
-				err := inout.ImportEventTypesJson(svixClient, reader, force)
+				err := inout.ImportEventTypesJson(context.Background(), svixClient, reader, force)
```

Comment:
For `import` and `export`, I'm using `Background` on purpose because having a timeout for 'bulk' operations might not be a good idea since they can take long with a large amount of data.

Reply:
Sure